mlflow/java/client/src/main/java/org/mlflow/client/objects/ObjectUtils.java

```diff
@@ -0,0 +1,34 @@
+package org.mlflow.client.objects;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import com.google.protobuf.MessageOrBuilder;
+
+import org.mlflow.api.proto.Service.*;
+
+import com.google.protobuf.util.JsonFormat;
+
+public class ObjectUtils {
```

Comment:
Why does this need to be public? If it's to expose the makeCreateRun here, I'd prefer to just add a createRun with parameters to the Client. Users will have a hard time finding this and it will be annoying to call.